cpp/include/cuml/fil/fil.h

```diff
@@ -126,23 +126,23 @@ struct sparse_node8_t : dense_node_t {
   sparse_node8_t(dense_node_t dn) : dense_node_t(dn) {}
 };
 
-/** leaf_value_t describes what the leaves in a FIL forest store (predict) */
-enum leaf_value_t {
+/** leaf_algo_t describes what the leaves in a FIL forest store (predict) */
+enum leaf_algo_t {
   /** storing a class probability or regression summand */
-  FLOAT_SCALAR = 0,
```

Comment:
Do we need to rename this? I think this is going to be used for binary classification and regression, and so should stay as is.